src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -214,8 +216,7 @@ class ItemPage extends React.Component {
 
         <div className="nypl-full-width-wrapper">
           <div className="nypl-row">
-            <div className="nypl-column-three-quarters nypl-column-offset-one">
-            </div>
+            <Search sortBy={sortBy} />
```

Comment:
shouldn't this be wrapped by a `<div className="nypl-column-three-quarters nypl-column-offset-one">`?